### DISCUSSION TITLE
Is it possible to combine location-based and marker-based ar?

### DISCUSSION THREAD

**Discussion by freds-dev** [AR-js-org/AR.js#292] [Help]
Here is what I want to do:
1. Scan a marker
2. All objects (saved wit lat,lon) render
3. I DONT want to use location-based because of its accuracy
4. I want to save the lat,lon of a marker and when its recognised I want to go from there

Do you know if that's possible? Thanks!

**Comment by nickw1** [comment]
So you use the marker with a known lat/lon for your original position, and then keep track of how far you move from the marker? Do you then use computer vision to measure the size of the marker, and use that to work out how far you have moved?  What about direction though? Will the marker have different patterns on its north, south, east and west edges?

I'd guess that may work if you do not move very far from the marker and it's still clearly visible. Other contributors who have worked on the marker-based side would know more.
